File: BasicForLoop.py
def main():
    someNumbers = [1,2,3,4,5]
    sumNumbers = 0
    # Using a for-loop, sum the numbers in someNumbers list
    for i in range(0, len(someNumbers)):
        sumNumbers += someNumbers[i]
    print(f"Sum of someNumbers: {sumNumbers}")


if __name__ == "__main__":
    main()

File: examples/part2-loops-and-conditionals/BasicForLoop.java
public class BasicForLoop {
    public static void main(String[] args) {

        int[] someNumbers = {1,2,3,4,5};
        int sumNumbers = 0;

        // Using a for-loop, sum the numbers in someNumbers array
        for (int i = 0; i < someNumbers.length; i++) {
            sumNumbers += someNumbers[i];
        }

        System.out.printf("Sum of someNumbers: %d\n", sumNumbers);
    }
}
